roms_tools/tests/test_setup/test_boundary_forcing.py

```diff
@@ -758,3 +761,60 @@ def test_from_yaml_missing_boundary_forcing(tmp_path, use_dask):
 
         yaml_filepath = Path(yaml_filepath)
         yaml_filepath.unlink()
+
+
+@pytest.mark.stream
+@pytest.mark.skipif(
+    not _has_dask(),
+    reason="Executed only if Dask package is installed",
+)
```

Comment:
To be consistent with the remaining code, we could consider removing the pytest markers above, and instead pass the fixture `use_dask` (defined in `conftest.py`) as an argument into the test function. Any strong opinions on your side?

Reply:
`use_dask` is nice (do we want to update it to have `skip` set?):

```py
@pytest.fixture(scope="session")
def use_dask(request: pytest.FixtureRequest):
    return request.config.getoption("--use_dask", skip=True)
```

I think the `stream` mark is really here to avoid letting these tests run on every execution, which is orthogonal to `use_dask`. That is, I would still like to avoid running tests that stream data (and take a "long time") even if `dask` is available.